src/dmd/mars.d

```diff
@@ -1751,6 +1751,8 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             params.verbose = true;
         else if (arg == "-vcg-ast")
             params.vcg_ast = true;
+        else if (arg == "-vasm") // https://dlang.org/dmd.html#switch-vasm
+            params.vasm = true;
```

Comment:
Why not put this in DMDparams, given that it is not a front-end option?

Reply:
I didn't know about DMDparams.